libs/libarchfpga/src/echo_arch.cpp

```diff
@@ -92,7 +92,7 @@ void EchoArch(const char* EchoFile,
                 VTR_ASSERT(false);
             }
             fprintf(Echo, " fc_value: %f", fc_spec.fc_value);
-            fprintf(Echo, " segment: %s", arch->Segments[fc_spec.seg_index].name.c_str());
+            //fprintf(Echo, " segment: %s", arch->Segments[fc_spec.seg_index].name.c_str());
```

Comment:
probably this is a leftover, but in general we shouldn't comment out code

Reply:
Removed commit that added this comment